internal/security/secretstore/tokenfilewriter/tokenfilewriter_test.go

```diff
@@ -32,17 +32,21 @@ import (
 	"github.com/edgexfoundry/go-mod-secrets/v2/secrets/mocks"
 )
 
-func TestNewTokenFileWriter(t *testing.T) {
-	lc, flOpener := setup()()
+var lc logger.LoggingClient
+var flOpener fileioperformer.FileIoPerformer
+
+func TestMain(m *testing.M) {
+	lc, flOpener = setup()()
```

Comment:
Why not move code from setup() into here? It is just a couple line that can be here.

Reply:
ok, i'll move it.

Reply:
fixed